src/mbgl/util/mapbox.cpp

```diff
@@ -50,6 +50,11 @@ std::string normalizeStyleURL(const std::string& url, const std::string& accessT
     }
 
     std::vector<std::string> pathname = getMapboxURLPathname(url);
+
+    if (pathname.size() < 3) {
+        return url;
```

Comment:
Log::Error(Event::ParseStyle, "Invalid URL for glyphs");

Would that be the appropriate kind of log?